lib/analytics.js

```diff
@@ -141,14 +141,21 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(settings, o
   }
 
   // initialize integrations, passing ready
+  // create a list of any integrations that did not initialize - this will be passed with all events for replay support:
+  this.failedInitializations = [];
```

Comment:
This becomes part of the Public API - so we should make sure we're on board with this name. At first glance `failedInitializations` as a name sounds weird to me. Though maybe this is ok in JS land?

Reply:
Definitely open to suggestions. Maybe `this.failedToInitialize`?

Reply:
`this.failureToLaunch`?

Reply:
I'm not too picky about the name itself, but since we are committing to it in the public API would  like to get a couple of folks to agree on the name.  I think we should go with whatever feels natural in the JS land.

`failedToInitializeIntegrations` sounds right to _me_ - however it's wordy and might not be ideal.

`failedInitializations` isn't that bad either. I'm ok with that if @tsholmes or @Peripheral1994 are as well.

Reply:
I'm fine with either. `failedInitializations` works well in the loopy land that is JS.

Reply:
how about `this.integrationInitFailures`?

Reply:
ok, let's stick to `failedInitializations` then!